q2_fmt/tests/test_engraftment.py

```diff
@@ -822,12 +823,13 @@ def test_get_subject(self):
                                         'total_donor_features', 'donor',
                                         'subject',
                                         'group'])
-        peds_df = _compute_peds(peds_df=peds_df, peds_type="Sample",
-                                peds_time=np.nan,
-                                reference_series=reference_series,
-                                table=table_df, metadata=metadata_df,
-                                time_column="group", reference_column="Ref",
-                                subject_column="subject")
+        peds_df = _compute_proportion(df=peds_df, type="Sample",
```

Comment:
```suggestion
        pedf_df = _compute_proportion(df=pedf_df, type="Sample",
```
same comment as above re: sample vs PEDF as type